### PR TITLE
fix(node): remove non-null assertion

### PR DESCRIPTION
**Pull request by jaredLunde**
The node ref can in fact be null at times and when it is it crashes apps. This replaces the non-null assertion with a conditional block.

### CODE REVIEW

File: src/node.tsx [hunk 1]
```diff
@@ -116,10 +116,11 @@ export class MathJaxNode_ extends React.Component<Props & MathJaxContextValue> {
     const inline = this.props.inline;
     const type = types[this.props.input];
     if (!this.script) {
```
Comment (captainsafia) [inline src/node.tsx]:
```suggestion
    if (!this.script && this.nodeRef.current) {
```

**Review by captainsafia — COMMENTED** [review]
Left a comment inline about condensing the if-statements.

What's the validation for this change?

**Comment by jaredLunde** [comment]
Thanks for the review! We (@Paperspace) have encountered situations where `this.nodeRef.current` can be `null`. This causes the component to throw with `cannot call appendChild() of null` errors. This PR fixes that issue for us. My guess is that [this line](https://github.com/nteract/mathjax/blob/243e89dc4a1aa79cf9b3531ebe4c92a530301dbd/src/node.tsx#L34) is the culprit. It's possible that React has already mounted and then unmounted the component by the time `setTimeout` is able to invoke `this.typeset()`. Lastly, since the type for `this.nodeRef.current` is `null | HTMLSpanElement` it's just a good practice to presume the type is correct and not assert otherwise IMO. In this case, the ref object is mutable so bugs are easy to sneak in.

**Comment by captainsafia** [comment]
@jaredLunde Published in @nteract/mathjax@4.0.14.

**Pull request merged**